frontend/src/host_orchestrator/orchestrator/controller.go

```diff
@@ -37,6 +37,7 @@ import (
 )
 
 const HeaderBuildAPICreds = "X-Cutf-Host-Orchestrator-BuildAPI-Creds"
+const HeaderUserProject = "X-Goog-User-Project"
```

Comment:
this header name is used by GCP, let's use our own name: `X-Cutf-Host-Orchestrator-BuildAPI-Creds-User-Project-ID`.